docs/devel/development.md

```diff
@@ -54,7 +54,7 @@ Kubernetes is written in the [Go](http://golang.org) programming language. If yo
 
 ### Go versions
 
-Requires Go version 1.4.x or 1.5.x
+Requires Go version 1.4.x, 1.5.x or 1.6.x
```

Comment:
That's not entirely true - currently we are running 1.4.2 and we officially do NOT support 1.5.X (but it is >= 1.4.2).

We need to explicitly specify versions that we are supporting.